phpBB/phpbb/db/tools/doctrine.php

```
@@ -363,6 +363,21 @@ function (Schema $schema) use ($table_name, $column): void
 		);
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	public function sql_truncate_table(string $table_name): void
+	{
+		try
+		{
+			$this->connection->executeQuery($this->get_schema_manager()->getDatabasePlatform()->getTruncateTableSQL($table_name));
+		}
+		catch (Exception $e)
+		{
+			return;
```

Just ignore the error?

---

No, I was thinking more in throwing an exception